src/main/flight/imu.c

```diff
@@ -534,8 +534,9 @@ static void imuCalculateEstimatedAttitude(float dT)
     bool useCOG = false;
 
 #if defined(USE_GPS)
+    const bool canUseCOG = isGPSHeadingValid();
```

Comment:
On MR we should probably check if we're more or less pitched forward, have significant ground speed and don't roll or yaw for a while to use CoG, this increases chances of GPS heading to be more or less aligned with true heading. WDYT?

Reply:
to be honest, right now I want to check how the transition from mag to cog will look like. And hopefully this will help a little with the MRs going sideways in CRUISE mode. test flight hopefully today